litellm/main.py

```diff
@@ -115,6 +116,8 @@ def completion(
     fallbacks=[],
     caching = False,
     cache_params = {}, # optional to specify metadata for caching
+    user_id: Optional[str] = None,
+    project = None,
 ) -> ModelResponse:
```

Comment:
we don't necessarily want to add a new param to `completion` that users not have to remember to use

Reply:
How can we update the budget without passing the "project" parameter?